src/app/systelab-components/numpad/README.md

```diff
@@ -5,7 +5,7 @@ Component to show a Numeric Keyboard dialog for an Input Text.
 ## Using the component
 
 ```html
-<systelab-numpad [numpadValue]="value" (numpadValueEmitter)="doSelectValue($event)"></systelab-numpad>
+<systelab-numpad [numpadValue]="value" (numpadValueEnter)="doSelectValue($event)"></systelab-numpad>
```

Comment:
Use the more standard [(value)] and (change)

Reply:
Done